Sources/SyncDataProviders/Bookmarks/BookmarksProvider.swift

```diff
@@ -82,7 +98,22 @@ public final class BookmarksProvider: DataProvider {
         let encryptionKey = try crypter.fetchSecretKey()
         context.performAndWait {
             let bookmarks = BookmarkUtils.fetchModifiedBookmarks(context)
-            syncableBookmarks = bookmarks.compactMap { try? Syncable(bookmark: $0, encryptedUsing: { try crypter.encryptAndBase64Encode($0, using: encryptionKey)}) }
+            syncableBookmarks = bookmarks.compactMap { bookmarkEntity in
+                do {
+                    return try Syncable(bookmark: bookmarkEntity, encryptedUsing: { try crypter.encryptAndBase64Encode($0, using: encryptionKey)})
+                } catch {
+                    if case Syncable.SyncableBookmarkError.validationFailed = error {
+                        os_log(
+                            .error,
+                            log: log,
+                            "Validation failed for bookmark %{public}s with title: %{public}s",
```

Comment:
Since title is user data, I believe it should be `%{private}`

Reply:
For some reason I thought `%{private}` wouldn't show up in Xcode console (while it's only true about Console.app) and I set it for debugging 🤦